content/applications/finance/accounting/receivables/customer_payments/batch.rst

```diff
@@ -2,34 +2,20 @@
 Batch payments: Batch deposits (checks, cash etc.)
 ==================================================
 
-A **Batch Deposit** groups multiple payments in a single batch. This allows you to deposit several
-payments into your bank account with a single transaction. This is particularly useful to deposit
-cash and checks.
+A **Batch Deposit** groups multiple payments in a single batch. This allows depositing several
+payments into your bank account with a single action. It can be advantageous to deposit any cash and
+check transactions rapidly.
 
-This feature allows you to list several customer payments and print a **deposit slip**. This ticket
-contains the details of the transactions and a reference to the batch deposit. You can then select
-this reference during a bank reconciliation to match the single bank statement line with all the
-transactions listed in the batch deposit.
+This feature lets you list several customer payments and print a deposit slip. It contains the
+details of the transactions and a reference to the batch deposit. You can then select this reference
+during a bank reconciliation to match the single bank statement line with all the transactions
+listed in the batch deposit.
 
 Configuration
 =============
 
 To activate the feature, go to :menuselection:`Accounting --> Configuration --> Settings -->
-Customer Payments`, activate **Batch Payments**, and click on *Save*.
-
-Payment Method Types
---------------------
-
-To record new payments as part of a Batch Deposit, you have to configure first the Journal on which
-you record them.
-
-To do so, go to :menuselection:`Accounting --> Configuration --> Journals`, open the Journal
-you want to edit, click on *Edit*, and open the *Advanced Settings* tab. In the *Payment Method
-Types* section, enable **Batch Deposit**, and click on *Save*.
-
-.. image:: batch/batch-journal-settings.png
-   :align: center
-   :alt: Payment Method Types in a journal's advanced settings
+Customer Payments`, activate :guilabel:`Batch Payments`.
 
 .. note::
    Your main bank accounts are automatically configured to process batch payments when you activate
```

Comment:
Not sure we should add this as a Note. If we speak about the Main bank accounts, what does it mean for the other ones?